wpimath/src/main/proto/trajectory.proto

```diff
@@ -15,6 +15,5 @@ message ProtobufTrajectoryState {
 }
 
 message ProtobufTrajectory {
-  double total_time = 1;
```

Comment:
This is calculated in the constructor, so it doesn't make sense to have in the message